shoryuken.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'dotenv'
 
-  spec.add_dependency 'aws-sdk-core', '> 2'
+  spec.add_dependency 'aws-sdk-core', '~> 2'
```

Comment:
@ota42y how about `=> 2`, so it would support 2 and 3?

http://guides.rubygems.org/patterns/#pessimistic-version-constraint

Reply:
No, shoryuken doesn't support aws-sdk-core 3.x and doesn't work 😢 
So we should lock 2.x.